rocAL/include/api/rocal_api_data_loaders.h

```diff
@@ -32,6 +32,31 @@ THE SOFTWARE.
  * \brief The AMD rocAL data loader functions.
  */
 
+extern "C" RocalTensor ROCAL_API_CALL rocalImageDecoder(
+    RocalContext p_context,
+    RocalTensor p_jpegs,
+    RocalImageColor rocal_color_format,
+    unsigned internal_shard_count,
+    bool is_output = false,
+    bool shuffle = false,
+    bool loop = false,
```

Comment:
Remove shuffle
Check where loop variable should be present either in reader / decoder